documentation/New_Joiner_Checklist.md

```diff
@@ -0,0 +1,49 @@
+# New Joiner Checklist
+
+Steps to complete during the candidates first week at Capco and the Digital Engineering team.
+
+## Prior to joining:
+
+-   Contact candidate to check what type of laptop they would like to have upon arrival
+-   Advise Raphaella on the requirements on the laptop for the new candidate
+-   Send candidate a welcome email advising them of where to go on the first day
+-   Pull candidate out of induction to briefly meet the team
+
+## Access rights and permissions:
+
+-   Get added to the Atlassian suite (Jira, Confluence)
+-   CircleCI permissions
+-   Bitbucket permissions
+-   Team SharePoint access
+-   Add team member to Engineering Trello Board
+
+## Team meetings and events
+
+-   Add the candidate to the team mail distribution list
+-   Send the series of weekly team meetings
+-   Add the candidate to the \#Digital\_Engineering Slack group
+-   Add the candidate to the the \#Digital\_Develop Slack group
+
+## General
+
+-   Read through the new joiner pack (link here: https://capcoinc-my.sharepoint.com/:f:/r/personal/stephen\_harrison\_capco\_com/Documents/Digital%20Develop%20Materials/New%20joiner%20pack?csf=1&e=fxtcOK)
+-   Ask Stephen Harrison to add the candidate to the Digital Develop the distribution list
+-   Ensure the candidate has assigned themselves to one of the Digital Engineering initiatives
+-   Download Slack
+-   Join the Digital Slack channel (CapcoDigital - [capcodigitalteam.slack.com](http://capcodigitalteam.slack.com))
+-   Ensure the candidate has a copy of the Signature, LinkedIn and Role document attached below
+-   Ask the candidate to link their StackOverflow page to the Capco page using this link: <https://stackoverflow.com/jobs/company/capco/join/27f9556c-857e-47b9-8b23-dbb060961c37>
+
+## Training:
+
+-   Complete your draft training plan and send to the training lead
+-   Start working through your chosen track within Pluralsight
+-   Complete Matts course 
+-   Complete Interview training (SC+)
+
+[![](https://api.media.atlassian.com/file/ae57e6f1-68b0-47f7-932c-9204c0729737/image?height=320&client=3baacb73-704b-4e08-9276-5c86982a139f&token=eyJhbGciOiJIUzI1NiJ9.eyJpc3MiOiIzYmFhY2I3My03MDRiLTRlMDgtOTI3Ni01Yzg2OTgyYTEzOWYiLCJhY2Nlc3MiOnsidXJuOmZpbGVzdG9yZTpmaWxlOmFlNTdlNmYxLTY4YjAtNDdmNy05MzJjLTkyMDRjMDcyOTczNyI6WyJyZWFkIl19LCJleHAiOjE1NTYxMDYwMTYsIm5iZiI6MTU1NjEwMjY1Nn0.SKWrQtAjZO5SdB42N7lwXxYDs86lnD1qmYLV9AhYMNU){height="250"}](/wiki/download/attachments/623247392/Role%20Title,%20Signature,%20LinkedIn.docx?version=1&modificationDate=1555408048676&cacheVersion=1&api=v2)
```

Comment:
Please review the token is open

Reply:
Thanks for spotting, will remove this file entirely as it is not required.